apps/remix-ide/src/app/components/preload.tsx

```diff
@@ -31,8 +31,10 @@ export const Preload = (props: any) => {
   )
 
   function loadAppComponent() {
+    console.log('loading remix in the preloader')
```

Comment:
please remove

Reply:
this is removed in https://github.com/ethereum/remix-project/pull/4897